### ISSUE TITLE
Pokestops

### ISSUE THREAD

**Issue by karimavine** [finndev/PokeBuddy#974]
can catch pokemons and fight gyms 
but i can`t collect pokestops does that mean im permban ?

**Comment by Skolzack** [comment]
No, you're softbanned. I'm softbanned more than 10hours and still. Niantic is fucking us all

**Comment by karimavine** [comment]
well hopefully  they don`t turn it into perm ban
not gonna do all that all over again lol

**Comment by 404unknown** [comment]
#912
